benchmarks/map_test.go

```diff
@@ -1,4 +1,4 @@
-package benchmarks
+package benchmarks // nolint: goimports
```

Comment:
As far as I know, there's no case ever wherein we'd want to use nolint for goimports. Please correct me if I'm wrong

Reply:
thought i'd removed them all